partner_contact_personal_information_page/README.rst

```diff
@@ -0,0 +1,91 @@
+======================================
```

Comment:
Migration of this one is already managed in https://github.com/OCA/partner-contact/pull/788

Reply:
Yes, I migrated both
This branch is rebased on that one